frontend/src/main/java/puf/frisbee/frontend/model/Team.java

```diff
@@ -46,4 +51,22 @@
 	 * Sets the current score the team has achieved.
 	 */
 	void setTeamScore(int score);
+
+	/**
+	 * Assignes player to existing team
+	 *
+	 * @param player2 second player in team
+	 * @param teamName of the team to be joined
+	 * @return true if joining was successful
+	 */
+	boolean joinTeam(Player player2, String teamName);
```

Comment:
Ich bin mir nicht sicher, ob ich grad einen Knoten im Gehirn hab... aber joined nicht jeder Player für sich? Also joinTeam(Player currentPlayer, string teamName) setzt dann (später) einen Aufruf mit dem aktuellen Player an das Backend und gibt true zurück, wenn der join successful war (und setzt dann hier im Model den teamName)?

Reply:
Bin mir nicht sicher... Ich hatte mir das so vorgestellt: 1. Player erstellt Team und ist dann automatisch im Team (also als player1 gespeichert) und Join ist dann nur für den 2. Player (also player2 im Team gespeichert). Hab da aber auch gerade einen Knoten im Gehirn. Keine Ahnung, wie wir das am besten machen...

Reply:
Im Prinzip kann man ja selbst ein Team erstellen oder joinen (wenn ein Player mehrere Teams haben kann) 🤔 ... vllt sollten wir auch createTeam aufrufen und dort nur den Namen in der DB speichern und danach joinTeam (mit dem jeweiligen Player)? Dann ist es getrennt, im Frontend dann:
Klick auf createTeam-Button -> Aufruf von createTeam und joinTeam 
Klick auf joinTeam -> Aufruf von joinTeam

Reply:
Achso, damit man joinTeam dann gleich nutzen kann, um das Team zu wechseln. Jetzt check ichs :smile:

Reply:
Würde auch denken, dass man jederzeit (natürlich nur innerhalb der StartView über das Dropdown-Menü) mit joinTeam das Team Wechseln kann, sofern es im Backend ein weiteres Team mit nur einem Player gibt. Dazu muss man allerdings schon auch wissen, wie dieses Team dann heißt. Alternativ müsste man eine Liste mit den noch unvollständigen Teams anbieten … das wäre mir jetzt aber zu aufwendig. Die Option createTeam sollte aber auch immer verfügbar sein, d.h. man kann einfach ein Team im Backend anlegen, sich dann aber entscheiden in einem anderen Team zu spielen. Warum nicht …

Reply:
Ach so, … ja richtig: Wünsche Euch ein frohes und erfolgreiches neues Jahr!